alica_engine/src/engine/parser/PlanWriter.cpp

```diff
@@ -522,7 +519,15 @@ string PlanWriter::getRelativeFileName(string file) {
         if (file.substr(file.size() - 4, 4) == ".beh" || file.substr(file.size() - 4, 4) == ".pty" ||
                 file.substr(file.size() - 4, 4) == ".pml" || file.substr(file.size() - 3, 3) == ".pp") {
             supplementary::SystemConfig* sc = supplementary::SystemConfig::getInstance();
-            string tfile = (*sc)["Alica"]->get<string>("Alica.PlanDir");
+            string tfile = (*sc)["Alica"]->get<string>("Alica.PlanDir", NULL);
+            supplementary::FileSystem::combinePaths(tfile, file);
```

Comment:
This line does nothing. The combinedPath is the returned string.

Reply:
good spot, let's fix this in the planwriter overhaul